lib/cms/hippo_importer/base.rb

```diff
@@ -0,0 +1,99 @@
+module Cms
+  module HippoImporter
+    class Base
+      def initialize(data:, docs:, to: 'en', parser: HippoXmlParser, html_decoder: HTMLEntities.new)
+        @records = parser.parse(data, [hippo_type])
+        @_site = to || 'en'
```

Comment:
I had the same questions. I just move the old code to a other location that I thought that was better. I will change that too.